Shared/Protocols/Objectable.swift

```diff
@@ -180,6 +180,28 @@ extension Objectable where Self: PFObject {
         return object
     }
 
+    static func fetchAndUpdateLocalContainer(where identifiers: [String],
```

Comment:
This fetches and updates the local store. Sometimes a user was out of date when first loading.